src/features/discord/functions/dev-prefix/get-discord-dev-prefix.ts

```diff
@@ -0,0 +1,39 @@
+import _ from "lodash";
+import { IGetDiscordDevPrefix } from "../../interfaces/dev-prefix/get-discord-dev-prefix";
+
+/**
+ * @param {Readonly<IGetDiscordDevPrefix>} config The configuration object
+ * [getDiscordDevPrefix#hasEmphasis]{@link IGetDiscordDevPrefix#hasEmphasis} will be set to true when not a boolean
+ * [getDiscordDevPrefix#asMention]{@link IGetDiscordDevPrefix#asMention} will be set to false when not a boolean
+ *
+ * @return {string} A string representing the current developer profile
+ */
+export function getDiscordDevPrefix(
+  config: Readonly<IGetDiscordDevPrefix>
+): string {
+  const hasEmphasis: boolean = _.isBoolean(config.hasEmphasis)
+    ? config.hasEmphasis
+    : true;
+  const asMention: boolean = _.isBoolean(config.asMention)
```

Comment:
isMention is better than asMention no ?

Reply:
@vvalembois I used "isMention" at first but when you really think about this, this is not a state, this is a display mod.
Like you would say "as portrait/landscape".
What do you think?